shell/platform/darwin/macos/framework/Source/FlutterPlatformNodeDelegateMac.mm

```diff
@@ -94,11 +104,11 @@
   __strong FlutterEngine* strong_engine = engine_;
   NSCAssert(strong_engine, @"Flutter engine should not be deallocated");
   NSRect ns_view_bounds =
-      [strong_engine.viewController.view convertRectFromBacking:ns_local_bounds];
-  NSRect ns_window_bounds = [strong_engine.viewController.view convertRect:ns_view_bounds
-                                                                    toView:nil];
+      [strong_engine.viewController.flutterView convertRectFromBacking:ns_local_bounds];
```

Comment:
Here as well: we should either be asserting that the view is loaded, or fizzling out/returning a zero rect if it's not.